python/pfs/drp/stella/constructFiberFlatTask.py

```diff
@@ -63,12 +65,16 @@ def combine(self, cache, struct, outputId):
         dataRefList = combineResults.dataRefList
         outputId = combineResults.outputId
 
+        # Get detector. Assume detector is the same for all input visits
+        detector = dataRefList[0].get('raw_detector')
+
         # Coadd exposures taken with the same slit dither position to remove CRs
         dithers = defaultdict(list)
         for dataRef in dataRefList:
             value = dataRef.getButler().queryMetadata("raw", "dither", dataRef.dataId)
             assert len(value) == 1, "Expect a single answer for this single dataset"
             dithers[value.pop()].append(dataRef)
+
```

Comment:
Don't introduce non-functional lines unless they're in a separate commit.

Reply:
Fixed.